api/src/main/resources/application.properties

```diff
@@ -55,6 +55,10 @@ app.passwordResetExpiryMinutes = 1440
 
 app.worms.restapi.url=https://www.marinespecies.org/rest
 
+# RLS
+
+app.rls.api.url=http://rls.tpac.org.au
```

Comment:
Also needs to be in `api/src/test/resources/application.properties`

Reply:
Ohh!! that's where I needed it. I didn't realise there was a second properties file...